src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/UnprocessableHttpErrorsTest.java

```diff
@@ -30,8 +30,8 @@ void should_return_false_for_a_4XX_retryable_error() {
     }
 
     @Test
-    void should_return_true_for_a_401_retryable_error() {
-        assertTrue(UnprocessableHttpErrors.isNonRetryableError(HttpStatus.UNAUTHORIZED));
+    void should_return_false_for_a_401_retryable_error() {
```

Comment:
could we rename this appropriately? I.e. should_return_false_for_a_401_non_retryable_error